dune

```diff
@@ -1 +1,11 @@
 (dirs :standard \ node_modules)
+
+(subdir
+ dist
+ (rule
+  (alias vscode)
+  (mode
+   (promote (until-clean)))
+  (action
+   (copy %{project_root}/src/vscode_ocaml_platform.bc.js
+     ./vscode_ocaml_platform.js))))
```

Comment:
This rule fails if the `dist` directory doesn't exist:
```
max /Users/max/Projects/vscode-ocaml-platform [0] $ dune build @vscode                                                                                                 promote-vscode-ext
Error: dist/vscode_ocaml_platform.js: No such file or directory
Done: 0/0 (jobs: 0)%                
```

Reply:
That's a dune bug actually :/

Reply:
I'm pretty sure it's going to cause the CI to fail, so it needs a workaround.

Reply:
Okay, I just created the dune file in dist. I would have preferred to ignore all of `dist` but we need to fix the dune bug first.